packages/app-store/routing-forms/api/responses/[formId].ts

```diff
@@ -63,15 +66,34 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     throw new Error("formId must be provided");
   }
 
+  const session = await getSession({ req });
```

Comment:
Earlier there was no way for a formId to be known by someone other than the creator of the form. Because now we are listing forms for the teams as well, it is a must now to ensure that the user has the right access to download the responses.